scripts/benchmark_java_project.js

```diff
@@ -0,0 +1,150 @@
+            runOnFunctionList(executable, program.models, program.arguments,
+                program.methodList, typeof program.stopAfterSymex != 'undefined');
+        } else {
+            console.log(run(executable, program.models, program.arguments,
+                program.functionName, program.classFile, typeof program.stopAfterSymex != 'undefined'));
+        }
+    })
+    .parse(process.argv);
+
+/// Add the content of json to args
+function addToArgs(args, json) {
+    const newArgs = Object.keys(json);
+    for (let i = 0; i < newArgs.length; i++) {
+        args[newArgs[i]] = json[newArgs[i]];
+    }
+    return args;
+}
+
+/// Convert the args to a string of the form "--arg-name arg-value ..."
+function argsToString(args) {
+    let string = "";
+    const keys = Object.keys(args);
+    for (let i = 0; i < keys.length; i++) {
+        if (typeof args[keys[i]] === "boolean") {
+            if (args[keys[i]])
+                string += ` --${keys[i]}`;
+        }
+        else if (Array.isArray(args[keys[i]])) {
+            for (let j = 0; j < args[keys[i]].length; j++)
+                string += ` --${keys[i]} "${args[keys[i]][j]}"`;
+        }
+        else {
+            string += ` --${keys[i]} "${args[keys[i]]}"`;
+        }
+    }
+    return string;
+}
+
+/// Get the coverage info from the output json output if available
+function readCoverageInfo(stdout) {
+    const jsonOutput = JSON.parse(stdout);
+    for (let i = 0; i < jsonOutput.length; i++) {
+        if (typeof jsonOutput[i].goalsCovered != 'undefined')
+            return { goalsCovered: jsonOutput[i].goalsCovered, totalGoals: jsonOutput[i].totalGoals }
+    }
+}
+
+function readJsonFile(fileName) {
+    return JSON.parse(execSync("cat " + fileName).toString());
+}
+
+/// Read the options from tools.json and launch the executable
+function run(executable, modelsPath, argumentsFile, functionName, classFile, stopAfterSymex) {
+    const config = readJsonFile(
+        (typeof argumentsFile != 'undefined') ? argumentsFile : "benchmark_java_arguments.json");
+    const args = addToArgs({}, config);
+    if (typeof modelsPath != 'undefined')
+        args.classpath = `${modelsPath}:${args.classpath}`;
+    if (typeof functionName != 'undefined')
+        args['function'] = functionName;
+    if (stopAfterSymex) {
+        args['show-vcc'] = true;
+        args['json-ui'] = false;
+        args.verbosity = 0;
+    }
+    // timeout isn't a jbmc option but only used by this script
+    args['timeout'] = false;
+    commandLine = `${executable} ${classFile} ${argsToString(args)}`
+    const startTime = new Date();
+    try {
+        const timeCommand = "export TIME=\"%U\"; /usr/bin/time -o tmp_time.out "
+        + commandLine; 
+        const output =
+            execSync(timeCommand, { timeout: 1000 * config.timeout }).toString();
+        let execTime = fs.readFileSync('tmp_time.out', 'utf8').split('\n')[0];
```

Comment:
The unit here seems to be seconds, whereas execTime below (:tiger2:) is in milliseconds. Should be made consistent.